skrub/_datetime_encoder.py

```diff
@@ -284,16 +346,57 @@ def fit_transform(self, column, y=None):
                 f"Column {sbd.name(column)!r} does not have Date or Datetime dtype."
             )
         if self.resolution is None:
-            self.extracted_features_ = []
+            self._partial_features = []
```

Comment:
It seems like `self._partial_features` and `self.extracted_features_` are the same thing. could we keep only extracted_features?